### DISCUSSION TITLE
Using the automated builds from github actions

### DISCUSSION THREAD

**Discussion by LeeZ900** [andrei-drexler/ironwail#344] [Q&A] [answered]
Probably dumb question but can these win64 builds be used for general purpose playing Quake or are they built just for dev purposes? Thank you for your work

**Comment by garoto** [comment]
Yes, they are regular optimized (-O2) builds, just like a released version would be.

**Reply by LeeZ900** [reply]
Thanks for answering! Was very helpful

**Comment by andrei-drexler** [comment, marked as answer]
They can indeed be used for playing, but I'd still recommend switching to the official release when it comes out.

**Reply by LeeZ900** [reply]
Will switch in a heartbeat whenever it comes! :)

**Reply by LeeZ900** [reply]
Btw Ironwail is a kickass name long live Ironwail!
